src/uniparc/components/entry/Entry.tsx

```diff
@@ -78,7 +84,45 @@ const Entry: FC = () => {
         </ErrorBoundary>
       }
     >
-      <EntryMain transformedData={transformedData} xrefs={xrefDataObject} />
+      <Tabs active={match.params.subPage}>
+        <Tab
+          title={
+            <Link
+              to={(location) => ({
+                ...location,
+                pathname: `/uniparc/${match.params.accession}/${TabLocation.Entry}`,
+                hash: undefined,
+              })}
+            >
+              Entry
+            </Link>
+          }
+          id={TabLocation.Entry}
+        >
+          <EntryMain transformedData={transformedData} xrefs={xrefDataObject} />
+        </Tab>
+        <Tab
+          title={
+            <Link
+              to={(location) => ({
+                ...location,
+                pathname: `/uniparc/${match.params.accession}/${TabLocation.FeatureViewer}`,
+                hash: undefined,
+              })}
```

Comment:
Would be good to use the `LocationToPath` config for uniparc and add a `subPage` as we do here:
```
  [Location.UniProtKBEntry]: `/${Namespace.uniprotkb}/:accession/:subPage?`,
```

Reply:
Good catch. Just added, all comments should now be addressed.